server/app/db/models/index.js

```diff
@@ -0,0 +1,36 @@
+import Fs from 'fs';
+import Path from 'path';
+import Sequelize from 'sequelize';
+import * as DbConfig from '../../../../config/db';
+
+const basename = Path.basename(module.filename);
+const env = process.env.NODE_ENV || 'development';
+const db = {};
+const config = DbConfig[env];
+let sequelize;
+
+if (config.use_env_variable) {
```

Comment:
config.use_env_variable is nolonger available in you config/db file, isn't this line is redundant ?